Game.java

```diff
@@ -64,6 +65,14 @@ public static void startNewGame() {
         game.pickGameMode();
     }
 
+    public Board createBoard(int size) {
+        String state = "";
+        for (int i=0; i < size * size; i++) {
```

Comment:
The Game shouldn't care about how the Board's state is represented.  This doesn't belong here it is initialization detail.